ETC.tsv

```diff
@@ -723,7 +723,7 @@ ETC_20150317_000722	Black Large Slime
 ETC_20150317_000723	Fire Golem
 ETC_20150317_000724	Warmed Up Gorgon Golem
 ETC_20150317_000725	Noble Tomblord
-ETC_20150317_000726	Blue Zolem
+ETC_20150317_000726	Blue Zolrem
```

Comment:
I changed it to keep things consistent. In Escanciu you find monsters called "Zolrem" which are refered to as Zolems, so I changed everything to "Zolrem". Should I change it all to "Zolem"?